frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -747,6 +794,13 @@
                         }
                     }
                     for (var i=0; i<vm.leaderboard.length; i++) {
+                        if (vm.leaderboard[i].submission__submission_meta_attributes == null){
+                            vm.IsLeaderboardHavingMetaAtts = false;
```

Comment:
replace with `vm.isLeaderboardHavingMetaAttributes`

Reply:
@KhalidRmb change this to `showSubmissionMetaAttributesOnLeaderboard` or something relevant. `isLeaderboardHavingMetaAttributes` doesn't sound right to me

Reply:
Done.